psutil/arch/freebsd/specific.c

```diff
@@ -1046,3 +1046,38 @@ psutil_sensors_cpu_temperature(PyObject *self, PyObject *args) {
         PyErr_SetFromErrno(PyExc_OSError);
     return NULL;
 }
+
+
+/*
+ * Return cpu 0 frequency information
+ */
+PyObject *
+psutil_cpu_freq(PyObject *self, PyObject *args) {
+    int current;
+    int max;
+    int core;
+    char sensor[26];
+    char available[1000];
```

Comment:
Maybe it makes more sense to rename this to `minmax` or something (assuming that's what the string represents, which I don't know).

Reply:
The string represents all available CPU frequency levels